src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -1816,10 +1817,11 @@ public static MoveEntityEvent createMoveEntityEvent(Cause cause, Transform<World
      * @param targetEntity The target entity
      * @return A new dismount ride entity event
      */
-    public static RideEntityEvent.Dismount createRideEntityEventDismount(Cause cause, Entity targetEntity) {
+    public static RideEntityEvent.Dismount createRideEntityEventDismount(Cause cause, Entity targetEntity, DismountType dismountType) {
```

Comment:
Could you add an `@Deprecated` duplicate method that defaults a `DismountType`? It's possible to do, as there are others included in this class.